docker/conf/images/conf-app-images.sh

```diff
@@ -1,26 +1,27 @@
+#!/bin/bash
+
 JAVA_VERSION=17.0.5
 JAVA_VERSION_X=${JAVA_VERSION}_8
 
 # https://hub.docker.com/_/eclipse-temurin/tags
 export JAVA_BASE_IMAGE=eclipse-temurin:${JAVA_VERSION_X}-jre
 
-export ASYNC_RECEIVER_IMAGE=async_receiver:1.0-SNAPSHOT
-export ASYNC_RECEIVER_JAR=AsyncReceiver-1.0-SNAPSHOT-spring-boot.jar
+# ------- image names -------
 
-export SYNC_RECEIVER_IMAGE=sync_receiver:1.0-SNAPSHOT
-export SYNC_RECEIVER_JAR=SyncReceiver-1.0-SNAPSHOT-spring-boot.jar
+export ASYNC_RECEIVER_IMAGE=jembi/jempi-async-receiver:latest
+export SYNC_RECEIVER_IMAGE=jembi/jempi-sync-receiver:latest
+export PREPROCESSOR_IMAGE=jembi/jempi-pre-processor:latest
+export CONTROLLER_IMAGE=jembi/jempi-controller:latest
+export EM_IMAGE=jembi/jempi-em-calculator:latest
+export LINKER_IMAGE=jembi/jempi-linker:latest
+export API_IMAGE=jembi/jempi-api:latest
```

Comment:
I think if we change these names it will break the local registry deploy. The docker-stack files expect those to be particular values. We may need a separate file with a separate set of exports for a platform image names.